Jenkinsfile.deploy

```diff
@@ -1,9 +1,21 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
 pipeline {
-    agent { label 'operator-slave' }
+    agent {
+        label 'kogito-operator-slave && !master'
+    }
+
+    // Needed for local build
+    tools {
+        maven 'kie-maven-3.6.2'
+        jdk 'kie-jdk11'
+    }
 
     parameters {
         string(name: 'DISPLAY_NAME', defaultValue: '', description: 'Setup a specific build display name')
+        
```

Comment:
can be removed?

Reply:
I'd rather keep parameters in "groups" for readability of it